### ISSUE TITLE
PushBullet gives the same location for every Pokemon it finds, and the location is far away from the set location in config.json.

### ISSUE THREAD

**Issue by fr0zendr4gon** [jxmorris12/PokemonGo-Finder#14]
![img_0769](https://cloud.githubusercontent.com/assets/20587502/17042186/9737bb60-4f79-11e6-8688-2ab09b360a09.jpg)
![img_0770](https://cloud.githubusercontent.com/assets/20587502/17042187/973a4b00-4f79-11e6-81e0-91498f6e9bc6.jpg)

## Current Behavior

PushBullet gives the same location for every Pokemon it finds, and the location is far away from the set location in config.json. All of the pokemons found seem to have very close or similar gps coordinates for some reason. 

## Steps to Reproduce (for bugs)

1. Get notification from PushBullet
2. Click on link to google maps in PushBullet
3. Same location on map appears for each pokemon found 
   4.
## Your Environment

- Version used: Current Version
- Environment name and version (e.g. Python 2.7): Python 2.7
- Operating System and version (desktop or mobile): Mac OS X adn iPhone iOS 9.3.3

**Comment by ldinatale1182** [comment]
Also seeing this behavior.

**Comment by fr0zendr4gon** [comment]
The coordinate that PushBullet gave me is:
28.0403275117, -82.467448603, 20z

When I typed the coordinates into google maps without the "20z" at the end,
google maps showed a location that's very close to the location that I set
in the config.json.
But when I added the "20z" back into the coordinates, google maps showed
the location that was very far away.
I think that the problem is with the 20z coordinates or the coordinates at
the end.
